qa/mixed-cluster/src/test/java/org/opensearch/backwards/PluginInfoIT.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.backwards;
+
+import org.opensearch.Version;
+import org.opensearch.client.Request;
+import org.opensearch.client.Response;
+import org.opensearch.cluster.metadata.IndexMetadata;
+import org.opensearch.common.settings.Settings;
+
+import org.opensearch.test.rest.OpenSearchRestTestCase;
+import org.opensearch.test.rest.yaml.ObjectPath;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class PluginInfoIT extends OpenSearchRestTestCase {
+    public void testPluginInfoSerialization() throws Exception {
+        // Ensure all nodes are able to come up
+        Response response = client().performRequest(new Request("GET", "_nodes"));
+        ObjectPath objectPath = ObjectPath.createFromResponse(response);
+        final Map<String, Object> nodeMap = objectPath.evaluate("nodes");
+        assertEquals(4, nodeMap.keySet().size());
```

Comment:
Can we validate the plugin version information more closely?

Reply:
Same test as PluginInfoIT in full-cluster-restart suite, detailed testing covered in follow up items - https://github.com/opensearch-project/OpenSearch/issues/12008